### PR TITLE
Move sensor material parameters to XML file

### PR DESCRIPTION
**Pull request by RomanTkachenko-dev**
### Briefly, what does this PR introduce?
Move sensor material parameters to XML file

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #772)
- [ ] New feature (issue #__)
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [ ] Changes have been communicated to collaborators

### CODE REVIEW

File: compact/pid/pfrich.xml [hunk 1]
```diff
@@ -88,6 +88,13 @@
 <constant name="AEROGEL_SEPARATOR_WALL_THICKNESS" value="0.05"/>
 <constant name="AEROGEL_OUTER_WALL_THICKNESS" value="0.1"/>
 
+<!-- HRPPD material definitions -->
+
+<constant name="PFRICH_HRPPD_WindowMat" value="Quartz"/>
+<constant name="PFRICH_HRPPD_PCBMat" value="G10"/>
+<constant name="PFRICH_HRPPD_MPDMat" value="SiliconDioxide"/>
+<constant name="PFRICH_HRPPD_ASICMat" value="SiliconCarbide"/>
```
Comment (veprbl) [inline compact/pid/pfrich.xml:96]: These values should not come from constants in `<define>`, but defined in xml inside the `<detector>` definition below.

**Comment by billlee77** [comment]
@wdconinc @kkauder @veprbl  Hi Wouter, Kolja, and Dimitry, Please take a 2nd look at the pull request, we made appropriate changes to define material differently,  could you review and provide further comments? Also, I am not sure why there is still a check that fails, and I am not sure it is related to our changes.

**Review by veprbl — APPROVED** [review]

**Comment by veprbl** [comment]
@billlee77 This is good. Thank you for addressing the comments!

**Comment by veprbl** [comment]
(the validate-material-map will fail and that's currently expected)

**Pull request merged**